tests/components/rfxtrx/test_switch.py

```diff
@@ -100,32 +101,29 @@ async def test_one_switch(hass, rfxtrx):
     )
     await hass.async_block_till_done()
 
-    hass.data[rfxtrx_core.DATA_RFXOBJECT] = rfxtrxmod.Core(
-        "", transport_protocol=rfxtrxmod.DummyTransport
+    state = hass.states.get("switch.test")
+    assert state
+    assert state.state == "off"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5802827)